nuget.config

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <clear />
+        <add key="github" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    </packageSources>
+    <packageSourceCredentials>
+        <github>
+            <add key="Username" value="%USERNAME%" />
+            <add key="ClearTextPassword" value= "%NUGET_KEY%" />
+        </github>
+    </packageSourceCredentials>
+</configuration>
```

Comment:
You need to update this so it looks like [one from one of the Apis](https://github.com/LBHackney-IT/tenure-information-api/blob/master/nuget.config)...